dask/dataframe/tests/test_groupby.py

```diff
@@ -3717,3 +3717,19 @@ def test_groupby_numeric_only_false(func):
         with ctx:
             pd_result = getattr(df.groupby("A"), func)()
         assert_eq(dd_result, pd_result)
+
+
+@pytest.mark.parametrize("func", ["var", "std"])
+@pytest.mark.parametrize("observed", [True, False])
+@pytest.mark.parametrize("dropna", [True, False])
+def test_groupby_var_dropna_observed(dropna, observed, func):
+    df = pd.DataFrame(
+        {
+            "a": [11, 12, 31, 1, 2, 3, 4, 5, 6, 10],
+            "b": pd.Categorical(values=[1] * 9 + [np.nan], categories=[1, 2]),
```

Comment:
Is the use of categoricals here meaningful?

Reply:
Yes, ``observed`` is categorical-specific, does not have an effect without categorically